client/components/telecom/telephony/group/number/feature/conference/telephony-group-number-feature-conference-configuration.html

```diff
@@ -199,18 +199,19 @@ <h2 data-translate="telephony_number_feature_conference_popover_title"></h2>
                 </div>
             </section>
 
-            <footer class="clearfix double-choice">
-                <button type="submit"
-                        class="btn btn-primary"
-                        data-ng-disabled="$ctrl.conferenceSettingsForm.$invalid || !$ctrl.numberCtrl.number.feature.hasChange()"
-                        data-translate="submit">
-                </button>
-
-                <button type="button"
-                        class="btn btn-default"
-                        data-translate="cancel"
-                        data-ng-click="$ctrl.onPopoverCancel()">
-                </button>
+            <footer class="clearfix">
+                <div class="btn-group pull-right">
```

Comment:
`pull-right` => `float-right`

Reply:
This is exactly the same thing, https://bootstrapdocs.com/v3.3.5/docs/css/#helper-classes-floats
Also, pull-right is use 61 other times in 57 files and this is without components